jvb/src/main/kotlin/org/jitsi/videobridge/EndpointK.kt

```diff
@@ -290,6 +290,29 @@ class EndpointK @JvmOverloads constructor(
         }
     }
 
+    /**
+     * @return the password of the ICE Agent associated with this
+     * {@link Endpoint}.
+     */
+    override fun getIcePassword(): String = iceTransport.icePassword
+
+    /**
+     * Sends a message to this {@link Endpoint} in order to notify it that the set of endpoints for which the bridge
+     * is sending video has changed.
+     *
+     * @param forwardedEndpoints the collection of forwarded endpoints.
+     */
+    override fun sendForwardedEndpointsMessage(forwardedEndpoints: MutableCollection<String>) {
```

Comment:
Why has this become `MutableCollection`?

Reply:
Same as above, I'll add a TODO